docs/examples/and.rst

```diff
@@ -42,18 +42,18 @@ formulation of a BQM for a Boolean gate in detail. This example briefly repeats
 of mathematically formulating a BQM while adding details on the underlying physical processes.
 
 A D-Wave quantum processing unit (:term:`QPU`) is a chip with interconnected qubits; for example,
-a D-Wave 2000Q has up to 2048 qubits connected in a :term:`Chimera` topology. Programming it
+an Advantage QPU has over 5000 qubits connected in a :term:`Pegasus` topology. Programming it
 consists mostly of setting two inputs:
 
 * Qubit bias weights: control the degree to which a qubit tends to a particular state.
 * Qubit coupling strengths: control the degree to which two qubits tend to the same state.
 
 The biases and couplings define an energy landscape, and the D-Wave quantum computer seeks
-the minimum energy of that landscape. Once you express your problem in a formulation\ [#]_
+the minimum energy of that landscape. Once you express your problem in a formulation\ [1]_
 such that desired outcomes have low energy values and undesired outcomes high energy values,
 the D-Wave system solves your problem by finding the low-energy states.
 
-.. [#] This formulation, called an :term:`objective function`, corresponds to the :term:`Ising`
+.. [1] This formulation, called an :term:`objective function`, corresponds to the :term:`Ising`
```

Comment:
Why switch to explicit numbering? Won't that make it harder to maintain in the future? Also inconsistent with the citation you added lower down.

Reply:
There's [this](https://docutils.sourceforge.io/docs/user/rst/quickref.html#footnotes):

```suggestion
.. [#ising] This formulation, called an :term:`objective function`, corresponds to the :term:`Ising`
```